tests/e2e/specs/add-blocks.js

```diff
@@ -59,10 +35,7 @@ describe( 'Add Blocks', () => {
 		await page.focus( '.block-editor-block-list__block [role="textbox"]' );
 		const ctaTitle = 'This is an example CTA title';
 		await page.keyboard.type( ctaTitle );
-		await saveDraft();
-		await page.reload();
```

Comment:
This works when testing manually. 

We could get a snapshot of the failed state by upgrading `@wordpress/scripts`

But that'd probably break other things.

This might be enough for the scope of this ticket.